tests/BenchmarkDotNet.IntegrationTests.Classic/Exporters/ApprovalTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections.Generic;
+using ApprovalTests;
+using ApprovalTests.Namers;
+using ApprovalTests.Reporters;
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Exporters;
+using BenchmarkDotNet.Exporters.Csv;
+using BenchmarkDotNet.Exporters.Json;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Tests.Mocks;
+using Xunit;
+
+namespace BenchmarkDotNet.IntegrationTests.Classic.Exporters
+{
+    [UseReporter(typeof(XUnit2Reporter))]
+    public class ApprovalTest
+    {
+        [Theory]
+        [MemberData(nameof(GetExporters))]
+        public void Exporter(IExporter exporter, string testName)
+        {
+            NamerFactory.AdditionalInformation = testName;
+
+            var logger = new AccumulationLogger();
+            exporter.ExportToLog(MockFactory.CreateSummary(config), logger);
+
+            Approvals.Verify(logger.GetLog());
+        }
+
+        private static List<object []> GetExporters()
```

Comment:
Here we could use [`TheoryData<IExporter, string>`](https://github.com/xunit/xunit/blob/master/src/xunit.core/TheoryData.cs).